stable/rabbitmq/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: rabbitmq
-version: 0.5.2
```

Comment:
@tompizmor since this is a breaking change, I think we should bump the minor version of the chart.